package.json

```diff
@@ -94,7 +111,10 @@
   },
   "license": "Apache-2.0",
   "scripts": {
+    "link-react-native": "react-native link react-native-randombytes",
```

Comment:
I'd like to avoid most of the changes in this commit altogether. AFAICT, all we do with the JWT token is decode it, which means we could switch to a simpler library like https://github.com/auth0/jwt-decode which doesn't require this huge change.